book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ Required ] I think it would be better if you create the `has-many` relationship side of classes `Book` and `Rental` (a book has many rentals). Currently, your `Book` class doesn't have this relationship.
- It would also be better if the `Book` class has a method for adding rentals. please see the example code for an example of how to achieve this.

## Example code.
```ruby
class Book
  attr_accessor :title, :author, :rentals

  def initialize(title, author)
    @title = title
    @author = author
    @rentals = []
  end

  def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' 
    # association. 
    Rental.new(date, self, person)
  end
```